e2e-tests/hardhat.config.ts

```diff
@@ -29,6 +29,7 @@ const config: HardhatUserConfig = {
       spec: "-",
       json: "test-results.json",
     },
+    forbidOnly: true,
```

Comment:
I want to avoid throwing an error on execution of the tests, as adding `.only` is a very common tool for running e2e tests locally. As such, I would prefer enforcement of `.only` should happen only at CI / lint time.

Have we looked into https://www.npmjs.com/package/eslint-plugin-mocha ?

Reply:
Yeah I simply pushed it behind if we could use what we have. I agree with your statement, let's add it to linting then.